modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -293,7 +293,7 @@ public boolean invoke(
 
             boolean branch = condition.test(e.toArray(new Entry[]{}));
```

Comment:
The variable name is strange, of course.

Reply:
Well, this code I did not touch here :) We could change it in another PR

Reply:
Ok